src/Story.php

```diff
@@ -4,6 +4,8 @@
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
+ *
+ * @method static Proxy get(string $name)
```

Comment:
I guess there is no solution to "type" this `Proxy`?

Reply:
I don't think so, a user would have to add a custom `@method`